chapters/09-package-website.Rmd

```diff
@@ -5,6 +5,77 @@
 - Getting all resources and material organized
 - Defining lifecycle of your package and individual functions (with lifecycle)
 
+## Learning Objectives
+
+* Describe the functionality `pkgdown` provides to deploy a website for package documentation
+* Customize the website with pre-existing themes
+* Organize the website with vignettes, tutorials and custom pages
+* Explain a functions' lifecycle process and the tools to handle it in R
+
+## Outline
+
+* `pkgdown`
+
+  * Text: What is `pkgdown`?
+  * Text: Benefits of having documentation be code-generated
+  * Text: Benefits of having a public website with the package docs
+  * Exercise: Build and serve the website locally
+
+* Website Customization
+
+  * Text: Why does making a package website standout is useful
+  * Text: Where do the `pkgdown` themes come from?
+  * Exercise: Change the theme of the website to one different from the default
+  * Text: Organizing beyond themes
+      * Text: Transition to talking about sections, tutorials and vignettes
```

Comment:
I feel like there needs to be an exercise here with the core package that does some organization.  That'll be easier to write once we know what the package looks like at this stage, but I'm imagining at least having learners recognize where the existing content in the package ends up on the packagedown site, and maybe a little renaming, reorganizing the nav, or organizing the reference page.

Reply:
I agree, it would be good to add an exercise about organizing the reference page in accordance with the functions' purposes.